cosmic-usage-ui/src/main/resources/public/app/main.js

```diff
@@ -72,6 +150,16 @@ const app = {
     generateReportButtonOnClick: function(event) {
         event.preventDefault();
 
+        this.generateSortableReport();
+    },
+
+    generateSortableReport: function() {
+        var currentSorting = this.getCurrentSorting()
```

Comment:
Even though the semi-column is optional (at the end of the line), we should use it IMO :)